### PR TITLE
Automation

### PR DESCRIPTION
**Pull request by pierre-ernst**
- [x] Better error handling
  - [a6bff4b](https://github.com/pierre-ernst/snyk-github-issue-creator/commit/a6bff4b22b53b166a34de23f2214f0e06268924d)
- [x] Display warning/confirmation when trying to log issue against a public repo
  - [eaa0a5a](https://github.com/pierre-ernst/snyk-github-issue-creator/commit/eaa0a5a08943b3f98aae20d83c59079bb58e31d7)
- [x] Support for snapshot URLs
  - [3578696](https://github.com/pierre-ernst/snyk-github-issue-creator/commit/35786964d6d61a369be11c2bd7f8488eb8a58a1a)
- [x] Prevent duplicates (subsequent  scans) 
  - [c636478](https://github.com/pierre-ernst/snyk-github-issue-creator/commit/c6364788bd346a5b0751907d0e5447b645d256a8)
- [x] Github Rate limiting and pagination
  - [b3b1a8a](https://github.com/pierre-ernst/snyk-github-issue-creator/commit/b3b1a8a18c15e44aa04ff43598bb5c8af97c0365)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "snyk-github-issue-creator": "./cli/index.js"
   },
   "engines": {
-    "node": ">=8"
+    "node": ">=12"
```
Comment (jportner) [inline package.json]: Is this change strictly necessary?
Just asking because Kibana is currently pinned to 10.x right now, so any Kibana devs that might want to use this repo will have to switch Node versions 😢

File: README.md [hunk 2]
```diff
@@ -1,29 +1,39 @@
...
-- The projectId is available via the [Snyk API](https://snyk.docs.apiary.io/#reference/projects/projects-by-organisation/list-all-projects).
-- The optional projetName allows to overrride the project name from Snyk (usefull when runing Snyk with CI/CLI integration)
+- You can retrieve your snykOrg Id from your org settings page on [Snyk](https://snyk.io) or via the [Snyk API](https://snyk.docs.apiary.io/#reference/organisations/the-snyk-organisation-for-a-request/list-all-the-organisations-a-user-belongs-to).
+- The SnykProject Id is available via the [Snyk API](https://snyk.docs.apiary.io/#reference/projects/projects-by-organisation/list-all-projects).
+- You need to either provide a valid `SnykOrg` Id and `snykProject` Id, or use the `stdin` option to parse the output of a snyk monitor command to retrieve the necessary parameters. 
+- The optional `projetName` allows to overrride the project name from Snyk (usefull when runing Snyk with CI/CLI integration)
```
Comment (jportner) [inline README.md]: Nit
```suggestion
- The optional `projectName` allows to overrride the project name from Snyk (useful when runing Snyk with CI/CLI integration)
```

File: cli/index.js [hunk 3]
```diff
@@ -1,189 +1,308 @@
 #!/usr/bin/env node
 
-const { prompt } = require('enquirer');
+const {Octokit} = require("@octokit/rest");
+const {throttling} = require("@octokit/plugin-throttling");
```
Comment (jportner) [inline cli/index.js]: 🙌🙌🙌

**Comment by jportner** [comment]
This is great stuff!

I noticed that f5b83a9 also fixed the indentation / formatting. Not sure what IDE/plugin you used for it -- might be good to use something like [Prettier](https://prettier.io/docs/en/index.html) and drop a config file for it in the repo.

File: package.json (hunk 1 above)
Comment (pierre-ernst) [reply package.json]: good point

**Comment by pierre-ernst** [comment]
> Prettier

will do, thanks for the suggestion
